release-testing/prepare.sh

```diff
@@ -60,25 +96,39 @@ conjur authn whoami
 }
 
 function register_conjur_admin_key() {
-  local CONJUR_ADMIN_API_KEY=$(exec_conjur \
-   conjurctl role retrieve-key "$(store_get CONJUR_ACCOUNT):user:admin")
+  local admin_user_id
+  admin_user_id="$(store_get CONJUR_ACCOUNT):user:admin"
```

Comment:
Why the difference here from other `local`s in other methods(all caps vs all lowercase)?